lesson5/hometask_TetyanaKarpova.rb

```diff
@@ -0,0 +1,22 @@
+class LazyHash < Hash
+  def lazy?(method_name)
```

Comment:
Strange var name . Yep, we have a method interface for every key, but `method_name` seems irrelevant to the current context. I would choose `key`, for example.